### PR TITLE
BaSP-MR-297: Header ,Navbar (Hamburguer menu) responsive

### PR DESCRIPTION
**Pull request by maurojjzz**
Add Loaders on Home, Sign Up and Login. Improve video height on home. Create Hamburguer menu component and implemented it on the project. Update some styles.

### CODE REVIEW

**Comment by maurojjzz** [comment]
[header/navbar responsive](https://trello.com/c/ya0rgrH4/297-header-navbar-responsive)

**Review by FrancoLelli — APPROVED** [review]

**Review by CristianLotorto — APPROVED** [review]
Great work Mauro!

File: src/Components/Header/header.module.css [hunk 1]
```diff
@@ -1,49 +1,156 @@
...
+  .brand{
+    display: none;
+  }
+}
+
+
+@media screen and (max-width: 430px) {
+  /* Aquí van los estilos específicos para móviles */
+}
+
```
Comment (YhanahiRosenthal) [inline src/Components/Header/header.module.css:155]: you should delete this comment

File: src/Components/HamburguerMenu/hamburguer.module.css [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+    flex-direction: row;
+    gap: 5px;
+    align-items: center;
+    justify-content: center;
+    font-size: 1rem;
+    text-decoration: none;
+    color: var(--primary);
+    transition: 0.2s color;
+    font-weight: 400;
+    transition: all 0.2s;
```
Comment (YhanahiRosenthal) [inline src/Components/HamburguerMenu/hamburguer.module.css:121]: you repeat prop transition

**Review by YhanahiRosenthal — APPROVED** [review]
Good job!

File: src/Components/HamburguerMenu/hamburguer.module.css (hunk 2 above)
Comment (maurojjzz) [reply src/Components/HamburguerMenu/hamburguer.module.css:121]: good catch

**Pull request merged**